docs/sources/articles/host_integration.md

```diff
@@ -65,11 +65,14 @@ a new service that will be started after the docker daemon service has started.
     [Unit]
     Description=Redis container
     Author=Me
+    Requires=docker.service
     After=docker.service
 
     [Service]
     Restart=always
     ExecStart=/usr/bin/docker start -a redis_server
+    # for more options, use 'run' instead of 'start', but not suggested
```

Comment:
Code needs to be in backticks (`). 

Also, this is not a complete sentence and does not make sense. What is not suggested?